deploy/README.md

```diff
@@ -284,6 +284,9 @@ REQUIRED_NUMBER_OF_VALIDATORS=1
 # correctly to the Foreign network.
 VALIDATORS=0x 0x 0x
 
+HOME_REWARDABLE=false
+FOREIGN_REWARDABLE=false
+
 # The flag defining whether to use ERC677BridgeTokenRewardable contract instead of
 # ERC677BridgeToken.
 DEPLOY_REWARDABLE_TOKEN=false
```

Comment:
Does it make sense to comment `DPOS_STAKING_ADDRESS ` and `DPOS_STAKING_ADDRESS ` in order to not allow user to make a mistake by initialising them in some value even if a rewardable token is not used.

Reply:
If these values are not required - it is a good idea to comment them out, from my point of view.

Reply:
yes, they are not required for the generic `erc20-to-erc20` bridge